### DISCUSSION TITLE
Lazy term scrolls to top when opened

### DISCUSSION THREAD

**Discussion by salloom-domani** [LazyVim/LazyVim#4408] [General]
This wasnt the case before the last update, I might've missed up the config but nothing near the lazyterm config, and thats why I havent opened an issue to make sure it's not my fault first.

Notice how it scrolls back to view when a key is pressed
[Screencast from 2024-09-06 16-55-46.webm](https://github.com/user-attachments/assets/f2f1aa82-3c32-40f5-9b86-6b6e4f3468e6)

**Comment by dpetka2001** [comment]
I don't use edgy, but from what I've tested this only happens with edgy and not without it. Without edgy it doesn't scroll to the top. So, if you want to create an issue do that at the edgy github repo, but I'm not sure whether this is intended behavior or not.

**Reply by salloom-domani** [reply]
What are good alternatives to edgy ?

**Reply by dpetka2001** [reply]
Sorry but no idea really, since this whole concept doesn't really intrigue me. I prefer opening tmux panes or just the default floating lazyterm. [toggleterm](https://github.com/akinsho/toggleterm.nvim) might be something you want to look into, but it's been such a long time since I last tried it, that I can't really make a suggestion from my part. Just try it out and see if it fits your needs or not.
